source/index.html.md

```diff
@@ -38,6 +38,7 @@ includes:
   - administration/workspace_settings
   - administration/custom_fields
   - administration/quotas
+  - administration/activity
   - administration/activity_log
```

Comment:
There seems to be an Activity section here already. Can these be merged?

Reply:
Let me see, probably